src/protocol/graphics.ts

```diff
@@ -238,20 +238,28 @@ export function setupGraphics(store: UIStore) {
     const pause = ThreadFront.currentPause;
     assert(pause);
 
-    const rv = await pause.repaintGraphics();
-    if (pause === ThreadFront.currentPause && rv) {
-      const { mouse } = await getGraphicsAtTime(ThreadFront.currentTime);
-      let { description, screenShot } = rv;
-      if (screenShot) {
-        repaintedScreenshots.set(description.hash, screenShot);
-      } else {
-        screenShot = repaintedScreenshots.get(description.hash);
-        if (!screenShot) {
-          console.error("Missing repainted screenshot", description);
-          return;
+    try {
+      const rv = await pause.repaintGraphics();
+      if (pause === ThreadFront.currentPause && rv) {
```

Comment:
A null rv means that repainting failed.  So, instead of throwing an exception we can just add logic that runs in the !rv case.